src/utils/temp_folder.test.ts

```diff
@@ -0,0 +1,23 @@
+import { expect } from 'chai';
+import { cleanUpTempFolder, getTempFolder } from './temp_folder';
+import * as fs from 'fs';
+
+describe('temp folder test', () => {
+	it('temp returns a folder that exists', () => {
```

Comment:
Add a nested describe here like:
```
describe('getTempFolder function', () => {
```
and put all the getTempFolder tests in it

Reply:
Then rename this test to
```
	it('returns a folder that exists', () => {
```
The net result of this is that the full test name is:
`temp folder functions getTempFolder function returns a folder that exists`